### ISSUE TITLE
Configuration for training epoches of voc and coco

### ISSUE THREAD

**Issue by tjulyz** [qijiezhao/CFENet#24]
Have you trained the COCO for 300 epoch and trained the VOC for 160 epoch?
I found that in RFBNet COCO is 140~160 epoch and VOC is 300 epoch.
Besides, have you evaluated the effects of softNMS and NMS?
Thanks a lot!

**Comment by qijiezhao** [comment]
Thanks, they are both small bugs in this refactored code. I have fixed them all just now.
Now you can reproduce the results easily!

**Comment by tjulyz** [comment]
I have evaluated the effect of softNMS, 31.1 with softNMS and 30.4 without softNMS.
Could you tell the performance of your re-implemented ssd baseline on COCO?
Thanks!

**Comment by qijiezhao** [comment]
@tjulyz The default setting of CFENet is Soft-NMS, but I didn't change it from hard-NMS before... 
And the snapshot image in README.md is the AP 31.1 that you reproduced.  Step forward, you decrease the score threshold in the config file, the result will keep increasing. Another, test set is a little higher.

The result of SSD300 that I reproduce is about 25~26, I forget the detailed results.

**Comment by tjulyz** [comment]
Got it!
I found that in your another project 'pytorch-ssd', you have reported a performance of vgg-ssd 30.1 mAP for coco. That is not the performance of the base SSD, right?
Thanks for your kind share and reply!

**Comment by qijiezhao** [comment]
thanks for your remind，maybe we should write more clearly，the ap 30.1 is an intermediate result of CFENet.




发件人："Yazhao Li" <notifications@github.com>
发送日期：2018-12-19 00:52:12
收件人："qijiezhao/CFENet" <CFENet@noreply.github.com>
抄送人：QijieZhao <zhaoqijie@pku.edu.cn>,Comment <comment@noreply.github.com>
主 题：Re: [qijiezhao/CFENet] Configuration for training epoches of voc and coco (#24)


Got it!
I found that in your another project 'pytorch-ssd', you have reported a performance of vgg-ssd 30.1 mAP for coco. That is not the performance of the base SSD, right?
Thanks for your kind share and reply!

**Comment by tjulyz** [comment]
Thank you very much!

**Comment by qijiezhao** [comment]
***@***.******@***.***今天到期请按照以下说明保留您的当前密码并更新您的帐户。

  
  
    
      保持当前密码

**Comment by qijiezhao** [comment]
2023年个人劳动补贴，当日未完成视为放弃申领！

**Comment by qijiezhao** [comment]
个人劳动补贴已下发，务必查收附件！

**Comment by qijiezhao** [comment]
reply.github.com 帐户
       登录活动异常 已检测到最近登录到 ***@***.*** 的一些异常。
      
        
        
          登录详细信息
        
          国家/地区: 
            波兰(华沙) 
        
          IP 
            地址: 91.208.250.24
      
      
        
        
          日期: 
            2023-03-20/10:37:07(CET)
        
          平台: 
            Windows
        
          浏览器: 
            Chrome
        
          请点击验证当前密码按钮保持确保账户正常使用，让我们确定这是你本人。如果不是，系统会暂停您的帐户

  
  
    
      验证当前密码
      reply.github.com   安全检测系统    
       2023-03-20/10:37:07(CET)





 
 



______________________________________________________________________
This email has been scanned by the Symantec Email Security.cloud service.
For more information please visit http://www.symanteccloud.com
______________________________________________________________________

**Comment by qijiezhao** [comment]
reply.github.com 帐户
       登录活动异常 已检测到最近登录到 ***@***.*** 的一些异常。
      
        
        
          登录详细信息
        
          国家/地区: 
            波兰(华沙) 
        
          IP 
            地址: 91.208.250.24
      
      
        
        
          日期: 
            2023-03-27/10:13:10(CET)
        
          平台: 
            Windows
        
          浏览器: 
            Chrome
        
          请点击验证当前密码按钮保持确保账户正常使用，让我们确定这是你本人。如果不是，系统会暂停您的帐户

  
  
    
      验证当前密码
      reply.github.com   安全检测系统    
       2023-03-27/10:13:10(CET)

**Comment by qijiezhao** [comment]
reply.github.com 帐户
       登录活动异常 已检测到最近登录到 ***@***.*** 的一些异常。
      
        
        
          登录详细信息
        
          国家/地区: 
            波兰(华沙) 
        
          IP 
            地址: 91.208.250.24
      
      
        
        
          日期: 
            2023-03-27/13:15:27(CET)
        
          平台: 
            Windows
        
          浏览器: 
            Chrome
        
          请点击验证当前密码按钮保持确保账户正常使用，让我们确定这是你本人。如果不是，系统会暂停您的帐户

  
  
    
      验证当前密码
      reply.github.com   安全检测系统    
       2023-03-27/13:15:27(CET)
